docker-compose.yml

```diff
@@ -16,17 +16,13 @@ services:
       - '15672:15672'
 
   eventstoredb:
-    image: eventstore/eventstore:latest
+    image: eventstore/eventstore:lts
```

Comment:
😂 BTW we'll have to deal about matrixes of testing versions at some point in our CI to ensure ongoing compatibility with what people have

Reply:
The issue is with docker-compose configuration not with our code. It's not related to PR. On the `main` branch it will fail also.